packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -110,8 +111,17 @@ export class VerticalStackedBarChartBase extends React.Component<
     this._handleMouseOut = this._handleMouseOut.bind(this);
     this._calloutId = getId('callout');
     this._tooltipId = getId('VSBCTooltipId_');
-    this._adjustProps();
-    this._dataset = this._createDataSetLayer();
+    if (
+      this.props.data &&
+      this.props.data.length &&
+      !this.props.data.filter((item: IVerticalStackedChartProps) => item.xAxisPoint === undefined).length
```

Comment:
why checking for undefined

Reply:
removed, we don't need it for checking if the chart is empty